src/transformers/onnx/utils.py

```diff
@@ -61,3 +64,32 @@ def compute_serialized_parameters_size(num_parameters: int, dtype: ParameterForm
         Size (in byte) taken to save all the parameters
     """
     return num_parameters * dtype.size
+
+
+def get_preprocessors(model_name: str) -> List[Union[AutoTokenizer, AutoFeatureExtractor, AutoProcessor]]:
+    """
+    Gets all preprocessors (tokenizer, feature extractor or processor) that are available for `model_name`.
+
+    Args:
+        model_name (str): Name of the model for which preprocessors are loaded.
+
+    Returns:
+        List[Union[AutoTokenizer, AutoFeatureExtractor, AutoProcessor]]: list of preprocessors
+    """
+    procs: List[Union[AutoTokenizer, AutoFeatureExtractor, AutoProcessor]] = []
```

Comment:
I don't think we need typing here right?
```suggestion
    procs = []
```